src/components/controls/controls.js

```diff
@@ -0,0 +1,218 @@
+var registerControls = require('../../core/controls').registerControls;
+var THREE = require('../../lib/three');
+
+var MAX_DELTA = 0.2; // ms
+var PI_2 = Math.PI / 2;
+
+/**
+ * Controls component.
+ *
+ * Receives input events from device-specific control components, and applies movement and rotation
+ * to the element accordingly.
+ */
+module.exports.Component = registerControls('controls', {
+  dependencies: ['velocity'],
+
+  schema: {
+    enabled: { default: true },
+
+    flyingEnabled: { default: false },
+
+    position: { default: ['gamepad-controls', 'keyboard-controls', 'touch-controls'] },
```

Comment:
![soon6](https://cloud.githubusercontent.com/assets/1848368/14338653/8c97f4e4-fc46-11e5-8045-ae7d69c3d0ca.jpg)

This CV1 shipping delay is rough though. :(

Anyway I would expect to do this with a `getPosition()` method on `hmd-controls` maybe, as opposed to the current `getVelocityDelta()`?